index.js

```diff
@@ -0,0 +1,40 @@
+// tooling
+import postcss from 'postcss';
+
+// plugin
+export default postcss.plugin('postcss-system-ui-font', opts => {
+	const familyOpt = Object(opts).family;
+
+	const systemUiFamily = typeof familyOpt === 'string'
+		? familyOpt.trim().split(/\s*,\s*/)
+	: familyOpt || [
+		'system-ui',
+		'-apple-system',
+		'BlinkMacSystemFont',
+		'Segoe UI',
+		'Roboto',
+		'Oxygen',
+		'Ubuntu',
+		'Cantarell',
+		'Fira Sans',
+		'Droid Sans',
+		'Helvetica Neue'
+	];
+
+	// system-ui and fallbacks match
+	const whitespace = '[\\n\\r\\x09\\x20]';
```

Comment:
Newline also includes `\f`, see https://www.w3.org/TR/css-syntax-3/#newline-diagram

Reply:
I’ll add this